core/src/main/java/org/elasticsearch/cluster/routing/allocation/command/AllocateEmptyPrimaryAllocationCommand.java

```diff
@@ -128,7 +130,8 @@ public RerouteExplanation execute(RoutingAllocation allocation, boolean explain)
         }
 
         initializeUnassignedShard(allocation, routingNodes, routingNode, shardRouting, unassignedInfoToUpdate, StoreRecoverySource.EMPTY_STORE_INSTANCE);
-
+        Logger logger = Loggers.getLogger(AllocateEmptyPrimaryAllocationCommand.class);
```

Comment:
I'd add this as a `static final` field in `AllocateEmptyPrimaryAllocationCommand`.